src/client/components/forms/relationship.js

```diff
@@ -141,7 +182,10 @@ class RelationshipEditor extends React.Component {
 			rowsSpawned
 		});
 	}
-
+	/**
+	* This function deletes all relationships which have not been selected.
```

Comment:
Hmm, not exactly. If you look at the code, a list of relationships is generated with unselected ones replaced with `null`. Then the `_reject` function will filter out the `null` values, leaving only the selected relationships, which are assigned to `relationshipsToDelete`. So the selected relationships are the ones being deleted. You should also mention what form event triggers this handler.